package.json

```diff
@@ -1,61 +1,57 @@
 {
   "name": "react-sortable-tree",
   "version": "2.1.0",
-  "description": "Drag-and-drop sortable component for nested data and hierarchies",
+  "description":
```

Comment:
I should probably ignore package.json in prettierrc. Can you discard unrelated changes? and/or put it in another PR Thanks :)